cyhy_commander/commander.py

```diff
@@ -760,6 +760,20 @@ def do_work(self):
             ),
         )
 
+        # clear out retrieved but unprocessed work
+        self.__logger.debug("Process any leftover local jobs")
```

Comment:
I prefer this verbiage to "leftover", but feel free to ignore:
```suggestion
        self.__logger.debug("Process any previously-unprocessed local jobs")
```